### ISSUE TITLE
Error in 3-alignTips phase

### ISSUE THREAD

**Issue by jlipovac** [marbl/verkko#186]
Hi,

I was running verkko 1.4.1 version and using duplex reads (>=10kb) as --hifi input (also providing UL reads >= 50kbp as --nano input).
I got the same error mentioned in the issue #185.

Here is the snakemake log output:
[2023-08-19T135407.106912.snakemake.log](https://github.com/marbl/verkko/files/12452065/2023-08-19T135407.106912.snakemake.log)

I reran it with verkko 1.4.0 and it is running normally. 
Any idea what could be an issue?

**Comment by skoren** [comment]
What's the contents in 3-alignTips folder and the output in 3-alignTips/splitONT.err?

**Comment by jlipovac** [comment]
I've already rerun verkko with 1.4.0 version and with the same output dir so I don't have the content of 3-alignTips folder from that moment. If I run verkko again soon I'll try with sh/bash replacing

**Comment by skoren** [comment]
What was your linux version/type? I think Ubuntu is usually the one where sh links to dash not bash.

**Comment by jlipovac** [comment]
It is Ubuntu 20.04.5 LTS

**Comment by skoren** [comment]
Yeah, I'm pretty sure then it's the same as #185 and is due to using sh not bash. I'll close this one and track it together with #185.
